ferrischat_webserver/src/invites/get_guild_invites.rs

```diff
@@ -0,0 +1,71 @@
+use actix_web::{HttpRequest, HttpResponse, Responder};
+use ferrischat_common::types::{InternalServerErrorJson, Invite, NotFoundJson};
+use ferrischat_macros::get_db_or_fail;
+
+/// GET api/v0/guilds/{guild_id}/invites
+pub async fn get_guild_invites(req: HttpRequest, auth: crate::Authorization) -> impl Responder {
+    let db = get_db_or_fail!();
+    let guild_id = get_item_id!(req, "guild_id");
+    let bigint_guild_id = u128_to_bigdecimal!(guild_id);
+
+    let member_query = sqlx::query!(
+        "SELECT user_id FROM members WHERE user_id = $1, guild_id = $2",
+        auth.0,
+        bigint_guild_id
+    )
+    .fetch_optional(db)
+    .await;
+
+    match member_query {
+        Ok(member_id) => {
+            if member_id.is_none() {
```

Comment:
what do you do if member is is not null?

Reply:
Nothing, this was just to check if they had permission to view these invites by making sure they were in the guild.